lib/wasm/WebAssemblyGenerator.js

```diff
@@ -403,14 +411,20 @@ class WebAssemblyGenerator extends Generator {
 		const nextTypeIndex = getNextTypeIndex(ast);
 
 		const usedDependencyMap = getUsedDependencyMap(module);
+		const invalidExports = new Set(
```

Comment:
`invalidExports` sounds confusing to me, these are the exports that are handled by Webpack now, right? Maybe something like `exportsToRemove`?